build.sbt

```diff
@@ -28,7 +28,7 @@ val CssParserVersion = "1.0.0"
 
 lazy val commonSettings = Seq(
   scalaVersion := scala212,
-  crossScalaVersions := Seq(scala211, scala212, "2.13.0-M3"),
+  crossScalaVersions := Seq(scala211, scala212),
```

Comment:
Travis is failing on the 2.13.x builds. I think we'll have to remove them from the matrix too.

Reply:
(sbt-travisci!)